webchat/auth/loginCard.html

```diff
@@ -0,0 +1,51 @@
+<!-- login card -->
+<login-card-content>
+  <img id="login-logo" src="/app/images/logowithname.png" />
+  <md-progress-circular md-mode="indeterminate" ng-show="ctrl.isLoadingUser()"
+    md-diameter="96">
+  </md-progress-circular>
+  <login-email-form ng-if="ctrl.email && !ctrl.isLoadingUser()">
+    <form class="login-form" ng-submit="ctrl.signIn()">
+      <md-input-container class="form">
+        <label>Email</label>
+        <input type="email" name="email" ng-model="ctrl.user.email" class="login-card-input" required="false"
+          aria-label="Email">
+      </md-input-container>
+      <md-input-container class="form">
+        <label>Senha</label>
+        <input type="password" name="password" ng-model="ctrl.user.password" class="login-card-input" required="false"
+          minlength="6" aria-label="Senha">
+      </md-input-container>
+      <login-email-form-submit>
+        <md-button type="submit" class="md-raised md-primary push-right" md-colors="{background: 'teal-500'}">
+          Acessar
+        </md-button>
+      </login-email-form-submit>
+    </form>
+  </login-email-form>
+
+  <login-container ng-if="ctrl.google && !ctrl.isLoadingUser()">
+    <md-button class="login-btn" ng-click="ctrl.loginWithGoogle()" md-colors="{background: 'red-500'}">
+      <md-icon class="login-google-icon" md-svg-src="/app/icons/google-icon.svg"></md-icon>
+      <span>ACESSE COM GOOGLE</span>
+    </md-button>
+  </login-container>
+
+  <login-container ng-if="ctrl.invite && !ctrl.isLoadingUser()">
```

Comment:
The user must already have an account to log in to chat, so it does not make sense to have the option to request an invitation.